### PR TITLE
[FAU-407] Initial preparation

### PR DESCRIPTION
**Pull request by amiut**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)

**What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Chore

**What is the current behavior?** (You can also link to an open issue here)
https://inpsyde.atlassian.net/browse/FAU-407

**What is the new behavior (if this is a feature change)?**
- Added PR template
- Update package name to `rrze-webteam/fau-studium`

**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

**Other information**:

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
-    "name": "inpsyde/fau-degree-program",
+    "name": "rrze-webteam/fau-studium",
```
Comment (tyrann0us) [inline composer.json:2]:
```suggestion
    "name": "rrze/fau-studium",
```

File: composer.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
         "ext-dom": "*",
         "ext-libxml": "*",
         "inpsyde/assets": "^2.8",
-        "inpsyde/fau-degree-program-common": "*",
+        "rrze-webteam/fau-studium-common": "*",
```
Comment (tyrann0us) [inline composer.json:19]: As per https://github.com/RRZE-Webteam/FAU-Studium-Common/pull/1#discussion_r1582594223.
```suggestion
        "rrze/fau-studium-common": "*",
```

File: package.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
 {
-  "name": "@inpsyde/fau-degree-program",
+  "name": "@rrze-webteam/fau-studium",
```
Comment (tyrann0us) [inline package.json:2]: Same here.
```suggestion
  "name": "@rrze/fau-studium",
```

**Review by tyrann0us — CHANGES_REQUESTED** [review]
Apart from the changes to `composer.json`, please update `composer.lock`. The QA checks currently fail because of this.

File: fau-degree-program.php
Comment (o-samaras) [inline fau-degree-program.php:1]: Let's rename this to match the package's name, or alternatively use `index.php`.

**Review by o-samaras — APPROVED** [review]

**Review by tyrann0us — APPROVED** [review]
LGTM. Thank you for working on this and incorporating the changes! 💪🏽

**Pull request merged**